### DISCUSSION TITLE
Limiting scope of various Glances widgets

### DISCUSSION THREAD

**Discussion by rdswords** [Lissy93/dashy#1408] [Help and Support]
Having read through the System Resource Monitoring portion of the widgets guide (https://dashy.to/docs/widgets/#system-resource-monitoring), I don't see any options listed for some of the larger widgets to limit the scope of the information they display.

For example, are there any options that can be set in the Dashy conf to only display:
-specific network interface(s) within gl-network-interfaces
-specific disk(s) within gl-disk-space

**Comment by gdeans** [comment]
I've had to edit the glances.conf file to eliminate a lot of the crap items, but I still have issues getting the filesystem/folder items to display or be provided through the glances query (fs section in glances.conf).

**Comment by CrazyWolf13** [comment]
One of the first results on google already brought me to a solution to your problem, have a look here:

https://github.com/nicolargo/glances/issues/304#issuecomment-30910689

Should be as simple as adding this into glances config file.

```
[iodisk]
hide=sda2,sda5

[network]
hide=lo,eth1
```
Of course you need to fill in your own network adapter names or disk mountpoints

**Reply by gdeans** [reply]
I've actually figured out that the problem stems from running glances in a container.  It doesn't have access to the physical devices or mountpoints, so I'm working on the issue with adding various volumes to the container.  It's also running on my NAS that's using md raid which doesn't help, but it's still a work in progress.
